v0/trigger.json

```diff
@@ -2755,22 +2755,7 @@
             "properties" : {
               "type" : {
                 "type" : "string",
-                "enum" : [ "PullRequest", "Push", "IssueComment" ]
```

Comment:
Discussed offline: The reason to remove the issue_comment is add a filter under PR itself. Though backward incompatible, this is only internally used so accounts will be updated post rollout.